CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 - Adds the `useNYPLBreakpoints` hook. This hook internally uses Chakra's `useMediaQuery` hook to get the current responsive media query breakpoint.
 
+### Updates
+
+- Updates the `Icon` component to include variants for `Facebook`, `Instagram`, `Tumblr`, `Twitter` and `YouTube`.
+- Updates the `Icon` component to include variants for `Building`, `Exit`, `Locator`, `Power` and `Settings`.
```

Comment:
The building svg preview looks empty. Is that just the preview?

Reply:
@EdwinGuzman That was just the preview.  I made a small adjustment and the preview looks fine now.